elastic-utils-lib/resources/docker/docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+services:
+
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch:7.5.2
+    container_name: cmr-es-elastic
+    environment:
+      - http.host=0.0.0.0
+      - transport.host=127.0.0.1
+      - ELASTIC_PASSWORD=kibana
```

Comment:
kibana? weird password.

Reply:
This is an internal password that nobody cares. :)